src/parser.ts

```diff
@@ -4,11 +4,14 @@ import glob from "fast-glob";
 import ts from "typescript";
 
 const NODES_DIR = path.resolve("packages", "nodes-base", "nodes");
+const LANGCHAIN_NODES_DIR = path.resolve("packages", "@n8n", "nodes-langchain", "nodes");
 
 async function getDisplayNames() {
   const [nodeFilepaths, versionDescriptionFilepaths] = await Promise.all([
     glob(path.resolve(NODES_DIR, "**", "*.node.ts")),
     glob(path.resolve(NODES_DIR, "**", "versionDescription.ts")),
+    glob(path.resolve(LANGCHAIN_NODES_DIR, "**", "*.node.ts")),
+    glob(path.resolve(LANGCHAIN_NODES_DIR, "**", "versionDescription.ts")),
```

Comment:
We're destructuring the first two arrays from `Promise.all()` for the first two glob calls, so I'd expect these additions will be ignored?

Reply:
My bad; I originally just commented out the first two items and kept the LC ones for testing and didn't realize that it's being deconstructed. Fixed & thanks for catching it.